test/functional/fixtures/live/test.js

```diff
@@ -48,7 +47,16 @@ class RunnerMock extends LiveModeRunner {
     }
 }
 
-function createLiveModeRunner (tc, src, browsers = DEFAULT_BROWSERS) {
+function createTestCafeInstance (opts = {}) {
+    return createTestCafe({
+        hostname: '127.0.0.1', port1: 1335, port2: 1336, experimentalProxyless: config.proxyless, ...opts,
```

Comment:
I guess we can remove the `hostname: '127.0.0.1', port1: 1335, port2: 1336` options.